src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -2277,7 +2283,8 @@ static int getSurplusQueuedTaskCount() {
             (pool = (wt = (ForkJoinWorkerThread)t).pool) != null &&
             (q = wt.workQueue) != null) {
             int p = pool.mode & SMASK;
-            int a = p + (int)(pool.ctl >> RC_SHIFT);
+            // use max to ignore transient negative
+            int a = p + Math.max((int)(pool.ctl >> RC_SHIFT), 0);
```

Comment:
Should the ignoring transient negative be done on `a` w/ similar approach of
https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/5338a697e9b9ce138cfdc97b9c7dfd417dd31e3b/src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java#L2980-L2981